src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -165,7 +165,7 @@ typedef struct
 
 static xfer_ctl_t xfer_status[MAX_EP_COUNT][2];
 
-static xfer_ctl_t* xfer_ctl_ptr(uint32_t epnum, uint32_t dir)
+static inline xfer_ctl_t* xfer_ctl_ptr(uint32_t epnum, uint32_t dir)
```

Comment:
@pigrew inline furnction since this is quick and called often. If you have time, please let me know if you are OK with other changes of fsdev as well

Reply:
Probably would be optimized out by the compiler, anyway, but looks fine.

Reply:
Yeah, attribute always inline will probably make it truely inline. I used to put those attr but removed since it may not be portable with different compiler.